src/optimizer_interface.jl

```diff
@@ -858,14 +884,16 @@ function is_solved_and_feasible(
     dual::Bool = false,
     allow_local::Bool = true,
     allow_almost::Bool = false,
+    additional_termination_statuses::Vector{TerminationStatusCode} = TerminationStatusCode[],
```

Comment:
I do not see a reason to force a Vector, we should at least allow tuples.

Reply:
I really want to minimise the number of input types we support. This is part of the MethodError principle. Is there a good argument not to restrict to `Vector`? The allocations here are minimal, and checking this status should be so minor in comparison to solving the problem.

Reply:
```suggestion
    additional_termination_statuses::Union{
        Vector{TerminationStatusCode},
        NTuple{N,TerminationStatusCode}} = (),
```

Reply:
The Method error should be very clear with this suggestion. No weir tuples would be allowed.
Allocations are minimal, but there is no reason for them. Minimal allocations in a loop become big allocations.

Reply:
Who's calling this in a loop? And if they are, they cache the vector once outside the loop.